app/routes/planet_routes.py

```diff
@@ -0,0 +1,49 @@
+from flask import Blueprint, jsonify, abort, make_response
+
+class Planet:
+    def __init__(self, id, name, description, has_moon=None):
```

Comment:
Since it appears that you expect `has_moon` to be a boolean value, the default valuewould be less surprising if it were either `True` or `False` (probably `False`). We used `None` with inventory in swap meet in order to act as sentinel value so we could tell whether the call had passed in a (mutable) inventory list. Here, we can use a boolean value directly as the default.